grakn-graql/src/main/java/ai/grakn/graql/internal/analytics/Analytics.java

```diff
@@ -174,6 +180,7 @@ public long count() {
         ComputerResult result = computer.compute(new DegreeVertexProgram(allSubtypes),
                 new MinMapReduce(statisticsResourceTypes, dataType));
         Map<String, Number> min = result.memory().get(GraknMapReduce.MAP_REDUCE_MEMORY_KEY);
+        LOGGER.info("MinMapReduce is done");
```

Comment:
Personally not a fan of "empty" log lines like this.  It's kind of like printing `Started` and `Stopped` at the beginning and end of each method. Without some parameter or some additional information these log lines are not as useful as I think they can be. Even just knowing the keyspace of the graph you operating on could be handy.